Sources/XCTest/XCAbstractTest.swift

```diff
@@ -26,6 +26,37 @@ public class XCTest {
         fatalError("Must be overridden by subclasses.")
     }
 
+    /// The `XCTestRun` subclass that will be instantiated when the test is run
+    /// to hold the test's results. Must be overridden by subclasses.
+    public var testRunClass: AnyClass {
```

Comment:
Starting in the version of `XCTest.framework` that ships with Xcode 7.3, this property has a type of `AnyClass?`. If we want to strictly follow Apple XCTest here, we should use that. Alternatively, I'm wondering if it would be reasonable to adjust this to have a type of `XCTestRun.Type` (or `XCTestRun.Type?`) which would be considerably Swiftier while (IMO) not breaking the intention of the original API.